src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -0,0 +1,61 @@
+package mate.academy.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table (name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany (fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @OneToMany 
```
Redundant. You shouldn't add FetchType. OneToMany and ManyToMany have FetchType Lazy by default